code/game/objects/items/stacks/medical.dm

```diff
@@ -423,7 +435,8 @@
 		to_chat(user, span_warning("[M] is dead! You can not help [M.p_them()]."))
 		return FALSE
 	if(iscarbon(M))
-		return heal_carbon(M, user, heal, heal)
+		M.adjustFireLoss(heal, TRUE) //there's other, infinitely better ways to heal brute damage.
```

Comment:
Why this instead of the heal carbon proc

Reply:
because I was unsure if heal_carbon was even needed, since adjust procs exist and are straight-forward for these things